src/openforms/submissions/models/submission_report.py

```diff
@@ -65,11 +66,21 @@ def generate_submission_report_pdf(self) -> str:
         :return: string with the HTML used for the PDF generation, so that contents
           can be tested.
         """
+        try:
+            appointment_id = self.submission.appointment_info.appointment_id
```

Comment:
Can you add an extra check to only target new style appointments:

See https://github.com/open-formulieren/open-forms/blob/master/src/openforms/appointments/core.py#L84

```py
    if not submission.form.is_appointment:
        raise NoAppointmentForm("Not an appointment form")
```